### PR TITLE
doc: wayland: update desktop app names

### PR DESCRIPTION
**Pull request by greg-fer**
Updated nRF Connect for Desktop app names to match the application. Dropped `nRF Connect` from the name and added `app` after. Reasons in NCD-1066.

### CODE REVIEW

File: applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst [hunk 1]
```diff
@@ -180,9 +180,9 @@ Testing
 After programming the application and all the prerequisites to your development kit, test the application by performing the following steps:
 
 1. |connect_kit|
-#. Connect to the kit with a terminal emulator (for example, `nRF Connect Serial Terminal`_).
-   You can also use the :guilabel:`Open Serial Terminal` option of the `Cellular Monitor`_ app to open the Serial Terminal.
-   Using the Cellular Monitor app in combination with the nRF Connect Serial Terminal shows how the modem responds to the different modem commands.
+#. Connect to the kit with a terminal emulator (for example, `Serial Terminal app`_).
```
Comment (wiba-nordic) [inline applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst:183]:
```suggestion
#. Connect to the kit with a terminal emulator (for example, the `Serial Terminal app`_).
```

**Review by wiba-nordic — APPROVED** [review]
Picked one nit, but looks good to me.

File: samples/cellular/lte_ble_gateway/README.rst [hunk 2]
```diff
@@ -131,7 +131,7 @@ Program the main controller as follows:
          west build --board nrf9160dk@1.1.0/nrf9160/ns
 
 #. Verify that the programming was successful.
-   To do so, use a terminal emulator, like nRF Connect Serial Terminal, to connect to the first serial port and check the output.
+   To do so, use a terminal emulator, like the Serial Terminal app, to connect to the first serial port and check the output.
```
Comment (peknis) [inline samples/cellular/lte_ble_gateway/README.rst:134]:
```suggestion
   Use a terminal emulator, like the Serial Terminal app, to connect to the first serial port and check the output.
```

**Review by peknis — APPROVED** [review]
Approved with a minor nit.

**Review by mia-ko — APPROVED** [review]

File: applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst [hunk 3]
```diff
@@ -180,9 +180,9 @@ Testing
...
+   You can also use the :guilabel:`Open Serial Terminal` option of the `Cellular Monitor app`_ app to open the Serial Terminal app.
```
Comment (divipillai) [inline applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst:184]:
```suggestion
   You can also use the :guilabel:`Open Serial Terminal` option of the `Cellular Monitor app`_ to open the Serial Terminal app.
```

File: samples/bluetooth/peripheral_uart/README.rst [hunk 4]
```diff
@@ -26,7 +26,7 @@ The sample supports the following development kits:
    * When used with :ref:`zephyr:nrf5340dk_nrf5340`, the sample might support the MCUboot bootloader with serial recovery of the networking core image.
 
 The sample also requires using a compatible application for `Testing`_.
-You can use the `nRF Connect Bluetooth Low Energy`_ or `nRF Connect for Mobile`_ applications (or other similar applications, such as `nRF Blinky`_ or `nRF Toolbox`_).
+You can use the `Bluetooth Low Energy app`_ or the `nRF Connect for Mobile`_ mobile app (or other similar applications, such as `nRF Blinky`_ or `nRF Toolbox`_).
```
Comment (divipillai) [inline samples/bluetooth/peripheral_uart/README.rst:29]:
```suggestion
You can use the `Bluetooth Low Energy app`_ or the `nRF Connect for Mobile`_ mobile app (or other similar applications, such as `nRF Blinky`_ or `nRF Toolbox`_).
```
should me avoid the mention of two mobile words here?

File: doc/nrf/links.txt [hunk 5]
```diff
@@ -607,34 +608,34 @@
 .. _`Getting started guide for nRF Asset Tracker for Azure`: https://docs.nordicsemi.com/bundle/nrf-asset-tracker-saga/page/docs/azure/GettingStarted/Index.html
 .. _`nRF Asset Tracker Memfault integration`: https://docs.nordicsemi.com/bundle/nrf-asset-tracker-saga/page/docs/memfault/Index.html
 
-.. _`nRF Connect Board Configurator`: https://docs.nordicsemi.com/bundle/nrf-connect-board-configurator/page/index.html
+.. _`Board Configurator app`: https://docs.nordicsemi.com/bundle/nrf-connect-board-configurator/page/index.html
```
Comment (divipillai) [inline doc/nrf/links.txt:611]: Shoutcut.txt and cert-flashing.txt,  would require an update as well.

**Review by divipillai — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**